### DISCUSSION TITLE
QRB Calculator?

### DISCUSSION THREAD

**Discussion by AndreasK79** [magicbug/Cloudlog#1404] [Show and tell]
I'm wondering if anyone is interested in something like this?

Right now I've set it to pop up when you use ALT-Q. It's very basic right now, but it is working. I've been thinking of automatically filling locator 1 from the active station profile. Another thing I had in mind, was to add the maidenhead overlay.

Here is a preview:
![image](https://user-images.githubusercontent.com/6977712/153723718-41e4b410-162b-46d2-b82c-9d528029ec63.png)

**Comment by dg9vh** [comment]
Looks nice and would be a good idea, I think!

**Comment by magicbug** [comment]
Sorry for not responding over weekend was contesting, yes! def would like this

**Comment by AndreasK79** [comment]
Thanks @dg9vh and @magicbug for the response. I've gotten the grid to work now:
![image](https://user-images.githubusercontent.com/6977712/154054562-e5392b4f-3587-404c-85fa-c46ff7f4c18e.png)
It needs more polishing and error handling, but it's getting there.

**Comment by AndreasK79** [comment]
It's ready for testing now. PR is active.

**Comment by AndreasK79** [comment]
Here is a screenshot from how it looks in the PR:
![image](https://user-images.githubusercontent.com/6977712/154215783-d38a6730-2da4-4f0e-a117-fe9654cac141.png)

**Comment by dg9vh** [comment]
For full functional test I suggest to check from each quadrant to each
other, means: 12 Tests (4 * 3).

Am Mi., 16. Feb. 2022 um 08:23 Uhr schrieb Andreas Kristiansen <
***@***.***>:

> Here is a screenshot from how it looks in the PR:
> [image: image]
> <https://user-images.githubusercontent.com/6977712/154215783-d38a6730-2da4-4f0e-a117-fe9654cac141.png>
>
> —
> Reply to this email directly, view it on GitHub
> <https://github.com/magicbug/Cloudlog/discussions/1404#discussioncomment-2186485>,
> or unsubscribe
> <https://github.com/notifications/unsubscribe-auth/ADKN5OWZDJQ6IQCW5M2QWRTU3NGHLANCNFSM5OHJM3JQ>
> .
> Triage notifications on the go with GitHub Mobile for iOS
> <https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675>
> or Android
> <https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.
>
> You are receiving this because you were mentioned.Message ID:
> ***@***.***>
>


-- 
Amateurfunk ist das, was uns verbindet! Bleiben wir in Kontakt!
DARC e.V., Ortsverband Q03, Neunkirchen

**Reply by AndreasK79** [reply]
You mean from each corner? Like AR00MM to AA00MM, AR00MM to RR00MM etc? What are you trying to check is working? Drawing of line? Correct distance? Correct bearing?

**Comment by dg9vh** [comment]
Only from weithin the quadrand (anywhere) and for checing bearing.

Andreas Kristiansen ***@***.***> schrieb am Mi., 16. Feb.
2022, 15:20:

**Reply by AndreasK79** [reply]
Oh right, well, it uses the same QRA-library as the header and distance in QSO entry. Since that was fixed a while ago, it should be correct here as well.
